stf/src/sgx.rs

```diff
@@ -216,10 +218,14 @@ impl Stf {
                 )
                 .dispatch_bypass_filter(sgx_runtime::Origin::root())
                 .map_err(|_| StfError::Dispatch("balance_set_balance".to_string()))?;
+                increment_nonce(&root);
```

Comment:
same here. `increment_nonce` should be moved out of the `match`statement and be executed whenever the call succeeded

Reply:
Here we have the same problem as above : The root account is only known within the match statement.

Reply:
Actually not the same problem as above.. Fixed.